deep-learning/_posts/2017-10-25-deep-rl-image-captioning.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: review
+title:  "Deep Reinforcement Learning-based Image Captioning with Embedding Reward"
+tags:   deep-learning CNN reinforcement caption
+author: Clement Zotti
+pdf:   "http://openaccess.thecvf.com/content_cvpr_2017/papers/Ren_Deep_Reinforcement_Learning-Based_CVPR_2017_paper.pdf"
+cite:
+  authors: "S. Xie, R. Girshick, P. Dollár, Z. Tu, K. He"
+  title:   "Deep Reinforcement Learning-based Image Captioning with Embedding Reward"
+  venue:   "Computer Vision and Pattern Recognition, 2017. CVPR 2017"
+---
+
+The task of the model is to generate image captioning for the MSCOCO dataset using reinforcement learning. The novelty of this work is in two part :
+- The decision making based on a policy $$p_\pi$$ and a value $$ v_\theta $$ network.
+- A training of $$p_\pi$$ and $$ v_\theta $$ based on an actor-critic algorithm with a visual-semantic embedding reward.
+
+## Model
+
+Their model is separate in three part, the policy network, the value network and the visual-semantic embedding.
+
+
+#### Policy network
+The policy is a combination of the VGG16 and an LSTM, it is used to predict the next action given the current state, as shown in figure 2.
+
+![](/deep-learning/images/deep-rl-caption/fig2.png)
+
+#### Value network
+The value network has the same combination as the policy network but they add an MLP used for the regression task. It is used to evaluate the next word given the previous sentence generated.
```

Comment:
"It is used to evaluate the next word given the **image and the previous words generated in the sentence**."